lua/lspcontainers/init.lua

```diff
@@ -1,5 +1,8 @@
 -- default command to run the lsp container
 local default_cmd = function (runtime, workdir, image)
+  if string.match(workdir, ":") then
```

Comment:
I don't think this is a bad approach - I'm wondering if there is any Lua magic we could achieve to actually detect the host OS arch type.

Reply:
I know you can do vim.fn.has('win32') but I don't If vim functions work inside plugin files.

Reply:
> I know you can do vim.fn.has('win32') but I don't If vim functions work inside plugin files.

They do, using `vim.fn.has('win32')` is used a lot in plugins.

Reply:
Ok I switched that piece out to use has("win32")

Reply:
Awesome - this looks great! Thank you!